docs/essentials/collections/commonlyUsedCollectionTypes.md

```diff
@@ -1,20 +1,22 @@
 
-The LINQ to Objects feature allows you to use LINQ queries to access in-memory objects as long as the object type implements [IEnumerable](https://dotnet.github.io/api/System.Collections.IEnumerable.html) or [IEnumerable<T>](https://dotnet.github.io/api/System.Collections.Generic.IEnumerable%601.html). LINQ queries provide a common pattern for accessing data; are typically more concise and readable than standard foreach loops; and provide filtering, ordering and grouping capabilities. LINQ queries can also improve performance.
+The LINQ to Objects feature allows you to use LINQ queries to access in-memory objects as long as the object type implements [IEnumerable](https://dotnet.github.io/api/System.Collections.IEnumerable.html) or [IEnumerable&lt;T&gt;](https://dotnet.github.io/api/System.Collections.Generic.IEnumerable%601.html). LINQ queries provide a common pattern for accessing data; are typically more concise and readable than standard foreach loops; and provide filtering, ordering and grouping capabilities. LINQ queries can also improve performance.
 
 ## Related Topics
 
 Title | Description
 ----- | -----------
-[Collections and Data Structures](Collections-and-Data-Structures.md) | Discusses the various collection types available in the .NET Framework, including stacks, queues, lists, arrays, and dictionaries.
-[Hashtable and Dictionary Collection Types](hashtableandDictionaryCollectionTypes.md) | Describes the features of generic and non-generic hash-based dictionary types.
-[Sorted Collection Types](sortedCollectionTypes.md) | Describes sorted collections performance and characteristics.
+[Collections and Data Structures](Collections%20and%20Data%20Structures.md) | Discusses the various collection types available in the .NET Framework, including stacks, queues, lists, arrays, and dictionaries.
```

Comment:
@cartermp Original filename had spaces, later changed to dashes. I'll need to update the links in all collections sub-topics.